games-emulation/stella/stella-5.1.3.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop gnome2-utils
+
+DESCRIPTION="Multi-platform Atari 2600 VCS Emulator"
+HOMEPAGE="https://stella-emu.github.io"
+SRC_URI="https://github.com/stella-emu/${PN}/releases/download/${PV}/${P}-src.tar.xz"
```

Comment:
what is that? 

They have "normal" github link we prefer in gentoo

`https://github.com/stella-emu/stella/archive/${PV}.tar.gz -> ${P}.tar.gz`

Reply:
Nothing wrong with that? They have explicitly created a release tarball and it's .tar.xz so it'll be smaller.

Reply:
yes, release tarball is preferred always.

Reply:
@a17r since which days?

Reply:
@chewi we do not count bytes ;)

Reply:
@chewi are you kidding, I hope?

Reply:
I'm going with a17r and chewi here barring a wider consensus to the contrary

Reply:
@Zlogene, no? It's a minor point but I don't like personal preferences being stated as policy. Point me to official documentation where it says we have to use GitHub archive URLs and I'll drop it. To be honest though, I have seen more arguments to the contrary. I do use those URLs when necessary myself but they have been known to break checksums in the past.

Reply:
For now I'm taking chewi's side since he's a member of the games project which maintains this package, but I would like to see if there's any official documentation and not just the will of whoever decides to pull this.

Reply:
There are no sides here, we have always packaged release tarballs and that has not changed just because some projects on Github don't provide those.

Reply:
@chewi how is that policy? Re-read again, the most of gentoo ebuilds are using archive tarballs, now you started to nitpick gz vs xz, which is totally madness

Reply:
@Zlogene: 1) many projects don't provide a release tarball, 2) some packagers may not be aware of their existence. both points can not make you derive a perefence in Gentoo for the archive URI even if there may be a sizeable result through grep.